File: src/com/developerphil/adbidea/action/StartAction.java
package com.developerphil.adbidea.action;

import com.developerphil.adbidea.adb.AdbFacade;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.Project;

/**
 * Created by pbreault on 9/28/13.
 */
public class StartAction extends AdbAction {

    public void actionPerformed(AnActionEvent e, Project project) {
        AdbFacade.startDefaultActivity(project);
    }
}

File: src/main/kotlin/io/github/raghavsatyadev/adbidea/action/StartAction.kt
package io.github.raghavsatyadev.adbidea.action

import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.Project
import io.github.raghavsatyadev.adbidea.adb.AdbFacade

class StartAction : AdbAction() {
    override fun actionPerformed(e: AnActionEvent, project: Project) =
        AdbFacade.startDefaultActivity(project)
}
